openapi.yaml

```diff
@@ -17697,6 +17730,26 @@ components:
           description: This keypair's secret key. **Only returned on key creation**.
           example: '[REDACTED]'
           readOnly: true
+        limited:
+          type: boolean
+          description: Whether or not this key is a limited access key. Will return `false` if this key grants full access to all buckets on the user's account.
+          example: true
+        bucket_access:
```

Comment:
My other comments about type and property descriptions also apply to this section.